docs/en/basic_configuration.md

```diff
@@ -73,6 +73,34 @@ action. This module provides the following engine:
 * `SiteTreePublishingEngine` - regenerates the static cache files after a page is published/unpublished. This makes use
 of both interfaces as object trigger refresh of other objects.
 
+#### Example:  Triggering a republish of the homepage after an update to a DataObject
+
+In order for the triggers to get picked up and executed, make sure to add the SiteTreePublishingEngine to your DataObject.
+```yaml
+Project\ORM\YourDataObject:
+  extensions:
+    - SilverStripe\StaticPublishQueue\Extension\Engine\SiteTreePublishingEngine
+```
+
+Then implement the StaticPublishingTrigger interface on your DataObject:
+
+```php
+class YourDataObject extends DataObject implements staticPublishingTrigger 
+{
+
+    public function objectsToUpdate($context) 
+    {
+        return HomePage::get()->First();  // return 1 or multiple publishable objects
```

Comment:
Just for my certainty, this can return an array or a single item?

Reply:
Yes, both.   I updated the comment.